PayPal/src/main/java/com/braintreepayments/api/PayPalInternalClient.java

```diff
@@ -122,4 +130,12 @@ public void onResult(JSONObject tokenizationResponse, Exception exception) {
             }
         });
     }
+
+    private String findPairingId(Uri redirectUri) {
+        String pairingId = redirectUri.getQueryParameter("ba_token");
+        if (pairingId == null) {
+            pairingId = redirectUri.getQueryParameter("token");
+        }
+        return pairingId;
```

Comment:
[Similar to iOS](https://github.com/braintree/braintree_ios/blob/83a5709a06bca8ceb3b48f842ebefd4016294047/Sources/BraintreePayPal/BTPayPalClient.swift#L326): we check for `ba_token` and fallback to `token` if possible.